### DISCUSSION TITLE
Really Basic Question - how to play a track from tidal using mopidy-tidal

### DISCUSSION THREAD

**Discussion by peadarforbes** [EbbLabs/mopidy-tidal#127] [General]
Hi, 

I had been using mopidy spotify until that API was sunset recently. I decided to switch over to mopidy-tidal and have it installed and running on my raspberry pi. 

I have a basic question - how do I play a song from tidal?

with Mopidy-spotify I had a command like:
mpc add <spotify track URI"

this would add the track at the spotify URI to the playlist and play it. 

What is the equivalent for mopidy-tidal?

Thanks!

**Comment by tehkillerbee** [comment]
Hello, I would expect the usual tidal track URI to work; "tidal:track:xxxxx:yyyyy:zzzzzbut I have never actually used mpc or ncmpcpp or similar to access. I will try to test and get back to you.

@2e0byo do you usually use mpc/ncmpcpp for playback or do you use the web frontend?

Edit: fixed the tidal:track typo - it is bedtime for me!

**Reply by 2e0byo** [reply]
I've only ever used the Web frontend.  But if a properly formatted tidal track hits us and we don't play it, that's a bug.  I'll add some testa to the integration tests (which use mpc) if someone can give me some examples which ought to work.

**Reply by 2e0byo** [reply]
Fwiw don't you mean `tidal:track:etc` ?

**Reply by tehkillerbee** [reply]
Yes, you are right. It's too late for me now :) 

I know that there are a few other mopidy-tidal users who use mpc in one way or another. Let's try to ask them for examples.
